src/portable/synopsys/dwc2/dwc2_type.h

```diff
@@ -32,6 +32,12 @@ typedef struct
   uint32_t  ep_fifo_size;
 }dwc2_controller_t;
 
+typedef struct
+{
+  int8_t out_ep;
+  int8_t in_ep;
+}ep_avaliable_count_t;
```

Comment:
We can create a struct: 

```suggestion
typedef struct {
    uint8_t out_ep;
    uint8_t in_ep;
    uint8_t ep_max_count;
}dwc_ep_config_t;
```